resources/views/individuals/partials/progress.blade.php

```diff
@@ -27,6 +27,10 @@
                 @endcan
                 <button class="secondary" name="publish" value="1"
                     @cannot('publish', $individual) @ariaDisabled @endcannot>{{ __('Publish page') }}</button>
+                @cannot('publish', $individual)
+                <p>{{ __('You must attend an orientation session and fill in all the required information before you can publish your page.') }}
+                </p>
+            @endcannot
```

Comment:
For all of the cases where this text would show up, it seems that it is describing the disabled publish button. It might help if the button was marked up with an aria-describedby pointing at the text.

Reply:
Great idea, @jobara!